pkgs/development/tools/rust/cargo-pgrx/default.nix

```diff
@@ -76,4 +76,9 @@ in
     cargoHash = "sha256-pnMxWWfvr1/AEp8DvG4awig8zjdHizJHoZ5RJA8CL08=";
   };
 
+  cargo-pgrx_0_14_1 = generic {
```

Comment:
Would you mind using the latest patch release 0.14.3 ?

Reply:
Upstream maintainer says there is no breaking changes in patch version.
If you prefer to pin the patch as well, then i agree that build pgrx extension should include the version in it.

Reply:
Should have waited a bit to post the previous comment :sweat_smile:
I tried using cargo-pgrx 0.14.3, but it fails with this error:

```
Error:
   0: The installed cargo-pgrx 0.14.3 is not compatible with the dependencies in ./Cargo.toml:
      pgrx = 0.14.1, pgrx-macros = 0.14.1, pgrx-sql-entity-graph = 0.14.1
      cargo-pgrx and pgrx library versions must be identical.
      help: cargo install cargo-pgrx --version 0.14.1 --locked

Location:
   src/metadata.rs:83
```

https://github.com/pgcentralfoundation/pgrx/blob/c0de15d29d2ef175f3b56b630f85af09448094e5/cargo-pgrx/src/metadata.rs#L83

Reply:
I had a conversation with one of the maintainers. I tried finding the thread today, but i really dont remember where it is.

The conversation was about when stability will come and the tldr was not in the near future.